.pre-commit-config.yaml

```diff
@@ -0,0 +1,10 @@
+#largely copied from https://ljvmiranda921.github.io/notebook/2018/06/21/precommits-using-black-and-flake8/
+repos:
+    - repo: https://github.com/pre-commit/pre-commit-hooks
+      rev: v2.4.0
+      hooks:
+          - id: flake8
```

Comment:
I would run Black before flake8, otherwise flake8 will catch a bunch of errors missed by Black